packages/neuron-wallet/src/controllers/anyone-can-pay.ts

```diff
@@ -62,7 +63,13 @@ export default class AnyoneCanPayController {
 
   public async sendTx(params: SendAnyoneCanPayTxParams, skipSign = false): Promise<Controller.Response<string>> {
     const txModel = Transaction.fromObject(params.tx)
-    const txHash = await new TransactionSender().sendTx(params.walletID, txModel, params.password, 1, skipSign)
+    const txHash = await new TransactionSender().sendTx(
+      params.walletID,
+      txModel,
+      params.password,
+      params?.skipLastInputs ?? true,
```

Comment:
I've checked that the original value of `skipLastInputs` is limited to `0 | 1`, but please check it again for caution.

Reply:
I adjusted the type because I felt it was ambiguous in some cases.
https://github.com/Lester-xie/neuron/blob/feat/ckb-asset-account-transfer-to-general-address/packages/neuron-wallet/src/services/transaction-sender.ts#L53
This caused yesterday's unit test error, and I fixed it later.

Reply:
I really understand this update, what I want to make sure is that this parameter was limited to be `0` or `1` and cannot be other numbers. If it was limited to `0 | 1` then it's safe to use `true | false`.

I've checked references of this method and only `0` or `1` was used. For caution, let's check it twice.

Reply:
I saw this question when I was doing it, and it was determined that only `0 | 1` were used. To be precise, only `1`was used. This value is often not passed, and it defaults to 0.

Reply:
Gotcha